roles/docker/templates/daemon.json

```diff
@@ -60,7 +60,7 @@
     "tlskey": "",
     "tlsverify": true,
     "userns-remap": "",
-    "add-registry": {{  docker_additional_registries|default([]) }},
-    "blocked-registries": {{ docker_blocked_registries|default([]) }},
+    "add-registry": {{ l_docker_additional_registries | default([]) }},
+    "blocked-registries": {{ l_docker_blocked_registries|default([]) }},
     "userland-proxy-path": "/usr/libexec/docker/docker-proxy-current"
 }
```

Comment:
@ganhuang https://github.com/projectatomic/atomic-system-containers/pull/65 removes the use of ``/etc/sysconfig/docker`` from the container.